setup.py

```diff
@@ -46,6 +46,8 @@
 
 module = Extension('confluent_kafka.cimpl',
                    libraries=[librdkafka_libname],
+                   include_dirs=[os.getenv('include')],
```

Comment:
These envs are a bit too generic, don't think we'll want to have them as default in setup.py.

Reply:
1. This env. variable was set in batch file, but your script never use it.
2. And, it's for the build-tool to search for your own header files (files in 'librdkafka'), without this the build-tool will fail to find them.